agents/reinforce.py

```diff
@@ -1,4 +1,5 @@
-from base import TorchAgent
+from ml import BaseEncoder
+from .common import TorchAgent
```

Comment:
Same issue as before with usage of relative imports

```suggestion
from agents.common import TorchAgent
```